packages/server/src/index.ts

```diff
@@ -45,6 +50,40 @@ app.get('/', (_, res: Response) => {
   res.render('index')
 })
 
+app.post('/mailgun_logging_webhook', multer().none(), async (req, res) => {
```

Comment:
Can we move this to `webhooks` as a function like `registerPassportEndpoints`?  This helps factor out code.